.github/ISSUE_TEMPLATE/iceberg_bug_report.yml

```diff
@@ -9,7 +9,8 @@ body:
       description: What Apache Iceberg version are you using?
       multiple: false
       options:
-        - "1.1.0 (latest release)"
```

Comment:
Maybe we should make this part of the `How to release` process? https://iceberg.apache.org/how-to-release/

Reply:
good point, I have a PR open for updating release process, looks like this and also REV API update needs to be added https://github.com/apache/iceberg-docs/pull/210